src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WidgetPageObject.java

```diff
@@ -64,6 +68,9 @@ public WidgetPageObject createMultiple() {
     return this;
   }
 
+  /**
+   * Create incorrect widget tag on the tested article
```

Comment:
It is not obvious for me. When I read eg. googleForm.createIncorrect, I don't know what the author defined as incorrect. It may be incorrect widget because it is eg. forbidden content OR because it is insecure OR because the tag is wrong OR because parameters within the tag are wrong OR because it creates widget that overlaps other parts of content. For that reason I'd rather put a javadoc with more information on what is going to be incorrect.